openedx/stanford/cms/djangoapps/contentstore/views/utilities/bulkupdate.py

```diff
@@ -0,0 +1,146 @@
+"""
+Views related to bulk update operations on course problems
+"""
+
+from django.core.exceptions import PermissionDenied
+from django.contrib.auth.decorators import login_required
+from django.http import HttpResponseNotFound
+
+from edxmako.shortcuts import render_to_response
+from student.auth import has_course_author_access
+from util.json_request import JsonResponse
+from xmodule.capa_base import CapaFields
+from xmodule.modulestore.django import modulestore
+
+from opaque_keys.edx.keys import CourseKey
+
+from cms.djangoapps.contentstore.utils import reverse_course_url
+
+from .tasks import bulk_update_problem_settings
+
+SHOW_ANSWER_OPTIONS = []
+for value in CapaFields.__dict__['showanswer'].values:
+    SHOW_ANSWER_OPTIONS.append(value['value'])
```

Comment:
Not a dealbreaker, but the more Pythonic approach here would be to use a list comprehension:

```python
SHOW_ANSWER_OPTIONS = [
    value['value']
    for value in CapaFields.__dict__['showanswer'].values
]
```

Reply:
Done